asserts/snapasserts/validation_sets_test.go

```diff
@@ -1117,3 +1117,22 @@ func (s *validationSetsSuite) TestValidationSetKeySliceCommaSeparated(c *C) {
 	valSets := snapasserts.ValidationSetKeySlice([]snapasserts.ValidationSetKey{"1/a/a/1", "1/a/b/1", "1/a/b/2", "2/a/a/1"})
 	c.Assert(valSets.CommaSeparated(), Equals, "1/a/a/1,1/a/b/1,1/a/b/2,2/a/a/1")
 }
+
+func (s *validationSetsSuite) TestValidationSetKeyComponents(c *C) {
+	valsetKey := snapasserts.NewValidationSetKey(*assertstest.FakeAssertion(map[string]interface{}{
+		"type":         "validation-set",
+		"series":       "a",
+		"authority-id": "b",
```

Comment:
Is it on purpose that authority-id and account-id both use `b` as value? It seems like it makes the check below a bit ambitious ?